src/sentry/uptime/endpoints/organization_uptime_stats.py

```diff
@@ -215,3 +231,48 @@ def _map_response_to_project_uptime_subscription_ids(
             subscription_id_to_project_uptime_subscription_id[subscription_id]: data
             for subscription_id, data in formatted_response.items()
         }
+
+    def _get_date_cutoff_epoch_seconds(self) -> float | None:
+        value = float(options.get("uptime.date_cutoff_epoch_seconds"))
+        return None if value == 0 else value
+
+
+def add_extra_buckets_for_epoch_cutoff(
```

Comment:
We can probably add a TODO to remove this in 90 days

Reply:
will add a todo, thanks.